tests/network/macspoof/conftest.py

```diff
@@ -211,4 +213,4 @@ def vms_without_mac_spoof(
         vm.start(wait=True)
 
     for vm in stopped_vms:
-        running_vm(vm=vm, wait_for_cloud_init=True)
+        vm.wait_for_agent_connected()
```

Comment:
Don't you need to start them before waiting for the condition?

Reply:
That is what is done already, see line 213 above.
Or did you meant something else?

Reply:
I missed that, thanks!